ONSdigital/sixteens
# CMD typography fixes for

### What

Typography broke CMD styling.
Why? - CMD filterable template is using markdown styling.
Items which were broken and should now be fixed with this PR are:
* Headings are tabbed in
* Subheadings top padding removed
* List styling no longer appearing 'oddly' tabbed

### How to review

1. Checkout this branch and the following for dp-fronted-renderer
2. Run the CMD tech stack
3. Check that CMD content pages look as they should
4. Run the standard tech stack
5. Check that Articles (+Neutral), Compendiums and Bulletins still follow the new typography rules

### Who can review

Anyone on the developer team, ideally Frontend Dev except me.
